frontend/src/views/Profile.vue

```diff
@@ -119,6 +122,13 @@ export default {
       this.api.reload(this.user).then((user) => this.api.reload(user.profile()))
     }
   },
+  methods: {
+    reloadUser() {
+      this.api.reload(this.user).then((user) => {
+        this.$store.commit('login', user)
```

Comment:
To make this more easily understandable, could you add another mutation called "updateUser" to
frontend/src/components/store/auth.js, and then use that here?
The mutation code (single line) will be duplicated, but what we're doing here isn't really a re-login, but an update of the user state. Maybe these two actions will have different behaviour in the future.